src/Framework/Debug/StateCollector/EnvironmentCollector.php

```diff
@@ -23,11 +22,8 @@ public function populate(StateInterface $state): void
     {
         $state->setTag('php', \phpversion());
 
-        if (
-            $this->container->has(DispatcherInterface::class) &&
-            $this->container->get(DispatcherInterface::class) instanceof SapiDispatcher
-        ) {
-            $state->setTag('dispatcher', 'sapi');
+        if ($this->container->has(DispatcherInterface::class)) {
+            $state->setTag('dispatcher', DispatcherInterface::class);
```

Comment:
I am not sure about this change yet

Reply:
What about 
```suggestion
            $state->setTag('dispatcher', $this->container->get(DispatcherInterface::class)::class);
```

Reply:
Removed:
- [spiral/http] Added return type `bool` to the method `emit` in `Spiral\Http\EmitterInterface` interface.

Changed `$state->setTag('dispatcher', DispatcherInterface::class);` to `$state->setTag('dispatcher', $this->container->get(DispatcherInterface::class)::class);`